tests/jballerina-unit-test/src/test/resources/test-src/isolated-workers/isolation_inference_with_start_action.bal

```diff
@@ -399,6 +399,69 @@ function testClientClassMethodIsolationInference() {
     assertFalse(isMethodIsolated(c3, "bar"));
 }
 
+function f17() {
+    _ = start invoke();
+}
+
+function f18() {
+    _ = start invoke2();
+}
+
+public isolated function invoke() {
+}
+
+public function invoke2() {
+}
+
```

Comment:
Shall we also add a test where the called function is non-isolated due to actually not being isolated (rather than not inferring because public)?